.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        julia-version: ["lts", "1", "pre"]
+        julia-version: ["lts", "1"]
```

Comment:
why do we remove `"pre"`?

Reply:
Something is broken again, check the failing test for the first commit.

Reply:
*nirgs* DiffEq again.

Hm, can we maybe add them in a form that they are allowed to fail?

Reply:
We had nightly tests running on a separate CI job before, maybe it should be brought back.

Reply:
Yes maybe we could reactivate that and use pre there.